phantombot/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  server:
+    image: ghcr.io/phantombot/phantombot:3.14.1.0@sha256:dbec9818e40f967ac5aee3abcac5a1857481cbbe6b35400f9a2fa8f1dc638df0
+    user: 0:900
```

Comment:
If possible, we want to avoid running services as `root`. I haven't looked at the phantombot Dockerfile to see how they build the image so we might be stuck with this. But we should check if it is possible to run this non-root as the umbrel user (`1000:1000`).